scss/statics/top/unlogined.static.scss

```
@@ -135,18 +129,13 @@ h6:not(.overwrite) {
 }
 
 .ease-out {
-    animation-timing-function: cubic-bezier(0, 0, .2, 1);
+    $cubic-bezier: cubic-bezier(0, 0, .2, 1);
+    animation-timing-function: $cubic-bezier;
```

Stylelint (17.13.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)